src/js/components/Nav/stories/CustomAnchor.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Anchor, Box, Grommet, Main, Nav } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
+
+const navItems = [
+  { label: 'HTML', href: '#' },
+  { label: 'JS', href: '#' },
+  { label: 'CSS', href: '#' },
+  { label: 'REACT', href: '#' },
+];
+
+const customTheme = deepMerge(grommet, {
+  anchor: {
```

Comment:
This seems orthogonal to Nav, more about customizing Anchor.

Reply:
yet, I think it will be a helpful example. I can remove it if you strongly think it should be placed elsewhere. (I didn't find a Custom Anchor example on Anchor.)